project/plugins.sbt

```
@@ -4,13 +4,13 @@ resolvers ++= Seq(
     url("https://dl.bintray.com/content/sbt/sbt-plugin-releases"))(Resolver.ivyStylePatterns)
  )
 
-addSbtPlugin("com.47deg"          % "sbt-microsites"  % "0.7.0")
-addSbtPlugin("com.eed3si9n"       % "sbt-unidoc"      % "0.4.1")
-addSbtPlugin("com.github.gseitz"  % "sbt-release"     % "1.0.8")
-addSbtPlugin("com.jsuereth"       % "sbt-pgp"         % "1.1.1")
+addSbtPlugin("com.47deg"          % "sbt-microsites"  % "0.9.4")
+addSbtPlugin("com.eed3si9n"       % "sbt-unidoc"      % "0.4.2")
+addSbtPlugin("com.github.gseitz"  % "sbt-release"     % "1.0.11")
+addSbtPlugin("com.jsuereth"       % "sbt-pgp"         % "2.0.0")
 addSbtPlugin("com.lucidchart"     % "sbt-scalafmt"    % "1.15")
```

I think this should be replaced with:
```suggestion
addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.0.5")
```

I didn't updated it because it will likely change the current format and it should be on a separate PR.